### DISCUSSION TITLE
Compartilhamento de salário/ofertas (Salary sharing thread)

### DISCUSSION THREAD

**Discussion by lucis** [DyegoCosta/trabalhando-remoto#198] [General]
Boa noite, pessoal!

Recentemente pensei que seria interessante termos, como comunidade, uma noção maior sobre os salários e as ofertas do mercado. Existem iniciativas semelhantes como levels.fyi ou Glassdoor, porém nada focado no cenário trabalhos remotos para quem está no Brasil.

Pensando nisso, joguei a ideia no grupo do Telegram e criei [esse Airtable](https://airtable.com/shrMBEjBivMpuXUtC) aqui. É um formulário simples que joga os dados em uma tabela pública. Podemos ver como funciona por agora.

O que acham? 

Fiquem a vontade pra quem quiser responder sobre salários ou ofertas que você já recebeu.

@DyegoCosta no futuro, tendo uma base de dados maior, talvez possa ser um material legal para estar na home.

**Comment by ericcoutinho** [comment]
Bacana. Já contribui!!
Acho uma iniciativa válida

Em ter, 20 de jul de 2021 21:24, Lucis ***@***.***> escreveu:

**Comment by alvesmog** [comment]
É interessante colocar outras opções de moeda também

**Comment by megurost** [comment]
Bem legal a iniciativa!
Creio que a coluna "Company URL/LinkedIn", que não está sendo preenchida (talvez para evitar identificação), poderia ser substituída pela meio em que foi conseguida a vaga ou recebida a oferta (por exemplo pelo linkedin, site da empresa, indeed, github, etc). Assim quem está buscando vagas poderia ter um norte de plataformas com boas oportunidades.

Outra ideia é alterar o formulário pra incentivar o detalhamento da vaga/emprego, como a principal tecnologia ou conhecimento necessário. Os últimos postados são mais específicos nesse sentido (Unity Game Developer ou Machine Learning Enginner), mas somente o "Software Developer" fica vago. Talvez se o formulário tiver um campo a mais, ou somente um label de exemplo já incentive a adição desse tipo de informação.

**Comment by lucis** [comment]
@alvesmog @megurost gostei das sugestões, adicionei lá!
